source/stylesheets/modules/_awards.scss

```diff
@@ -85,6 +85,7 @@
 
   .organization {
     color: lighten($text-color, 20%);
+    @include ellipsis(50%);
```

Comment:
Or was that me playing around with the effects of ellipsis? That's the problem with taking over pull requests :p